router-bridge/src/introspect.rs

```diff
@@ -123,6 +124,7 @@ mod tests {
             }",
             vec![DEFAULT_INTROSPECTION_QUERY.to_string()],
         )
+        .unwrap()
         .unwrap();
```

Comment:
Just unwrap()? Or should this be expect with a message? Or even handle/propagate the error?

Reply:
true, it's in a test but it might help people if they fail someday 😅